.devcontainer/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN pwsh -Command Install-Module posh-git -Scope AllUsers -Force; \
 ENV PATH "$PATH:/home/${USERNAME}/bin"
 
 # Can be used to override the devcontainer prompt default theme:
-ENV POSH_THEME="/workspaces/oh-my-posh/themes/jandedobbeleer.omp.json"
```

Comment:
@ehawman-rosenberg I missed this, but it seems like this wasn't the intention, right?

Reply:
Is that not what it was? I had changed it to `devious-diamonds.omp.yaml` and I thought I was putting it back.